extension/src/popup/views/ReviewAuth/index.tsx

```diff
@@ -351,10 +351,12 @@ const AuthDetail = ({
     async function getIsToken() {
       try {
         const transfers = [];
-        const isToken = await getIsTokenSpec({
-          contractId: rootJson.args.source,
-          networkDetails,
-        });
+        const isToken = !rootJson.args.source
```

Comment:
for both the invocation and sub invocation this should account for not having a source in the root args.